src/python/bot/fuzzers/libFuzzer/launcher.py

```diff
@@ -677,6 +677,31 @@ def get_printable_command(command, fuzzer_path, use_minijail):
   return engine_common.get_command_quoted(command)
 
 
+def maybe_use_mutator_plugin(target_name, extra_env, fuzzing_strategies,
+                             chroot):
+  """Decide whether to use a mutator plugin. If yes and there is a usable plugin
+  available for |target_name|, then add it to LD_PRELOAD in |extra_env|, add a
+  strategy to |fuzzing_strategies|, and add chroot bindings if |chroot| is not
+  None."""
+  if not do_mutator_plugin():
```

Comment:
nit: i prefer "if do_mutator_plugin" to be in main, along with other do_ functions. that way, this function is just use_mutator_plugin instead of maybe_.

Reply:
Changed it to do this but I don't like this pattern and I think we should refactor this code so that all strategy handling isn't done in main.

Reply:
+1